docs/src/main/sphinx/admin/properties-task.md

```diff
@@ -4,7 +4,7 @@
 
 - **Type:** {ref}`prop-type-integer`
 - **Restrictions:** Must be a power of two
-- **Default value:** The number of physical CPUs of the node, with a minimum value of 2 and a maximum of 32
+- **Default value:** The number of physical CPUs of the node, with a minimum value of 2 and a maximum of 32. Defaults to 8 in fault tolerant execution mode
```

Comment:
If this is true we should - wrap this, link to FTE docs, and add end of sentence .

Reply:
True:
https://github.com/trinodb/trino/blob/4db4f606e770a15e62063e9f7c0383ee657c6cca/core/trino-main/src/main/java/io/trino/server/ServerMainModule.java#L332-L334

Reply:
Hm .. can you check if this affects other properties as well @posulliv and just update them all at once and maybe even add a section in the FTE docs page for it.

Reply:
@mosabua the only property that changes its default in FTE mode is `exchange.compression-codec`. This one is already covered in the exchange properties docs.